src/components/Exercises/AttachedFilesTable/AttachedFilesTable.js

```diff
@@ -94,7 +95,8 @@ AttachedFilesTable.propTypes = {
   newFiles: ImmutablePropTypes.list,
   addFiles: PropTypes.func,
   HeaderComponent: PropTypes.func.isRequired,
-  RowComponent: PropTypes.func.isRequired
+  RowComponent: PropTypes.func.isRequired,
+  intl: PropTypes.shape({ formatMessage: PropTypes.func.isRequired }).isRequired
```

Comment:
I think you don't use function `intl.formatMessage()` here, so there is no need to check its existence in prop types. Instead, you use a `intl.locale`, so you can check this in here. I'm not sure about the type, but I think it is a string. To sum up, you probably wanted something like
```
intl: PropTypes.shape({ locale: PropTypes.string.isRequired }).isRequired
```